From d85ca8e83ea0537fd74d8bea3c358a3edddb87ac Mon Sep 17 00:00:00 2001
From: Daniel Schultz <d.schultz@phytec.de>
Date: Tue, 6 Aug 2024 11:26:55 +0200
Subject: [PATCH] Switch submodule common to github
​
The freedesktop.org URL is currently not available. Therefore, this
plugin won't build. Use the official GStreamer mirror on GitHub.
​
Signed-off-by: Daniel Schultz <d.schultz@phytec.de>
---
 .gitmodules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
​
diff --git a/.gitmodules b/.gitmodules
index a6b1eda..80ffbc5 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,3 +1,3 @@
 [submodule "common"]
 	path = common
-	url = git://anongit.freedesktop.org/gstreamer/common
+	url = https://github.com/GStreamer/common
-- 
2.34.1
